src/GoogleAnalytics/Client.php

```diff
@@ -16,6 +16,7 @@ class Client
     public const ACCOUNT_PROFILES_URL = 'https://www.googleapis.com/analytics/v3/management/accounts/~all/webproperties/~all/profiles';
     public const ACCOUNTS_URL = 'https://www.googleapis.com/analytics/v3/management/accounts';
     public const REPORTS_URL = 'https://analyticsreporting.googleapis.com/v4/reports:batchGet';
+    public const REPORTS_PROPERTIES_URL = 'https://analyticsdata.googleapis.com/v1beta/%s:runReport';
```

Comment:
endpoint na report properties !!! v beta verzi !!! - api je použité jen na tyto reporty, tak by to nemělo ničemu vadit